os/arch/arm/src/amebasmart/amebasmart_serial.c

```diff
@@ -1399,14 +1399,6 @@ int up_getc(void)
  ****************************************************************************/
 int up_putc(int ch)
 {
-	u32 CounterIndex = 0;
```

Comment:
Regardless of this PR,, this #else case (non USE_SERIALDRIVER) is unnecessary.
up_putc and up_getc are perfectly the same with #if case.

Reply:
I will check why is this condition added, if unnecessary i will remove in a separable PR.